### DISCUSSION TITLE
More advanced editor functions?

### DISCUSSION THREAD

**Discussion by aisuneko** [ppy/osu#14504] [Q&A] [answered]
As an Linux-based osu! player I'm currently making some beatmaps in lazer. Turns out that lazer's editor lacks some key functions such as multiple difficulties editing, specifying preview point and publishing. For some reasons I can't use stable on Wine or in my virtual machine. Will these functions be implemented in the future? If already implemented, how to use them? Thanks!

P.S. (nvm if for some reason we won't get publishing done until lazer replaces stable.)

**Comment by peppy** [comment, marked as answer]
Yes they are all on the list of things left to be done.

**Reply by aisuneko** [reply]
I got it, thanks!
